test/y2storage/dialogs/guided_setup/select_scheme_test.rb

```diff
@@ -39,21 +39,7 @@
   subject { described_class.new(guided_setup) }
 
   describe "skip?" do
-    before do
-      allow(Y2Storage::Dialogs::GuidedSetup).to receive(:allowed?).and_return(allowed)
-    end
-
-    context "when the Guided Setup is not allowed" do
-      let(:allowed) { false }
-
-      it "returns true" do
-        expect(subject.skip?).to eq(true)
-      end
-    end
-
-    context "when the Guided Setup is allowed" do
-      let(:allowed) { true }
-
+    context "since the Guided Setup is always allowed" do
```

Comment:
So the context makes no sense here. Just remove it. Contexts are not there to attach explanatory texts to the output. :wink: